src/main/java/io/anserini/ltr/StopWords.java

```diff
@@ -0,0 +1,426 @@
+package io.anserini.ltr;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class StopWords {
```

Comment:
I think `Stopwords` is lowercase `w` elsewhere in the repo? Let's be consistent.
Also, please document where you got this list from...